new.php

```diff
@@ -116,12 +119,24 @@
 	}
 }
 
-$patchesAppliedText = implode( ' ', $patchesApplied );
+$branchDesc = preg_replace( '/^origin\//', '', $branch );
+
+$wikiName = "Patch Demo (" . trim(
+	// Add branch name if it's not master, or if there are no patches
+	( $branchDesc !== 'master' || !$patchesApplied ? $branchDesc : '' ) . ' ' .
```

Comment:
Put the extra space inside the ternary, otherwise you will get  "( 123456)", and "(REL1.35 )"

Reply:
There's the `trim()` to fix that. (It's annoying to do this without it, because either "side" can be empty.)